vyper/vyper.py

```diff
@@ -331,23 +347,27 @@ def _find(self, key):
                 log.debug('{0} found in environment: {1}'.format(key, val))
                 return val
 
-        env_key = self._env.get(key)
-
+        env_key = self._find_insensitive(key, self._env)
+        log.debug(f'Looking for {key} in env')
+        log.debug(self._env)
         if isinstance(env_key, list):
-            parent = next((self._config[real_key] for real_key in self._config.keys() if real_key.lower() == key), None)
+            parent = self._find_insensitive(key, self._config)
             found_in_env = False
+            log.debug(f'Found env key parent {key}: {parent}')
```

Comment:
Please change the log message to `'Found env key parent {0}: {1}'.format(key, parent)` so it works on older Python versions.